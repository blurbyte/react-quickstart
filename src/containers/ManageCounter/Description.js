import styled from 'styled-components';

const Description = styled.p`
  font-family: 'OpenSans Light', Arial, Helvetica, sans-serif;
  font-size: 2.4rem;
  color: #444;
  line-height: 3.4rem;
  margin: 1rem 0;
`;

export default Description;
