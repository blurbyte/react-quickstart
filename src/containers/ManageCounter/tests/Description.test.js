import React from 'react';
import { shallow } from 'enzyme';

import Description from '../Description';

describe('<Description />', () => {
  it('should render parahgraph>', () => {
    const component = shallow(<Description />);
    expect(component.type()).toEqual('p');
  });
  it('should have a className attribute', () => {
    const component = shallow(<Description />);
    expect(component.prop('className')).toBeDefined();
  });
  it('should render given text', () => {
    const children = 'Test';
    const component = shallow(<Description>{children}</Description>);
    expect(component.contains(children)).toBeTruthy();
  });
});
