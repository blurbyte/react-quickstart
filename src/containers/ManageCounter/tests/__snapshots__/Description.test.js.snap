// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppWrapper /> should render correctly 1`] = `
.jEZwaP {
  font-family: 'OpenSans Light', Arial, Helvetica, sans-serif;
  font-size: 2.4rem;
  color: #444;
  line-height: 3.4rem;
  margin: 1rem 0;
}

<p
  className="jEZwaP"
/>
`;
